go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/guptarohit/asciigraph v0.5.5
 	github.com/hetznercloud/hcloud-go v1.37.0
 	github.com/pelletier/go-toml v1.9.5
```

Comment:
I think this line needs to be updated to `github.com/pelletier/go-toml/v2 v2.0.6`.

This happens automatically if I run `go mod tidy` locally.

Reply:
oh yeah. Good catch. Thanks. Fixed now